File: Models/Photo.cs
namespace PropertySphere.Models;

public record Photo
{
    public string? Id { get; set; }
    public string? RenterId { get; set; }
    public string? Caption { get; set; }

    public string? Description { get; set; }
    public required string ConversationId { get; set; }
    public DateTime UploadedAt { get; set; } = DateTime.UtcNow;
}

File: Models/photo.py
from dataclasses import dataclass
from typing import Optional


@dataclass
class Photo:
    conversation_id: str = ""
    renter_id: Optional[str] = None
    caption: Optional[str] = None
    description: Optional[str] = None
    id: Optional[str] = None
    
    @classmethod
    def collection_name(cls) -> str:
        return "Photos"
